packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/lib/page-props-factory/plugins/personalize.ts

```diff
@@ -16,12 +16,16 @@ class PersonalizePlugin implements Plugin {
         ? context.params.path.join('/')
         : context.params.path ?? '/';
 
-    // Get variant for personalization (from path)
+    // Get variant(s) for personalization (from path)
```

Comment:
You also need to modify personalize middleware for variant prefixes to be present in the path